src/pyscipopt/scip.pxd

```diff
@@ -284,6 +284,9 @@ cdef extern from "scip/scip.h":
     ctypedef struct SCIP_NLROW:
         pass
 
+    ctypedef struct SCIP_NLROW:
```

Comment:
You redefined `struct SCIP_NLROW`. Please remove this.

Reply:
Oops. Thanks.